app/controllers/concerns/api/v3/sync_to_user.rb

```diff
@@ -90,5 +92,13 @@ def sync_region_modified?
       return if process_token[:sync_region_id].blank?
       process_token[:sync_region_id] != requested_sync_region_id
     end
+
+    def time(method_name, &block)
+      raise ArgumentError, "You must supply a block" unless block
```

Comment:
Is this necessary? Can we just let the `yield` below blow up with a `Block not provided`?

Reply:
Oh, right, of course 👍🏼